data/src/main/kotlin/com/lightteam/modpeide/data/utils/commons/FileSorter.kt

```diff
@@ -28,7 +29,8 @@ object FileSorter {
 
     private val fileNameComparator: Comparator<in FileModel>
         get() = Comparator { first, second ->
-            first.name.compareTo(second.name)
+            // Case insensitive
+            first.name.toLowerCase(Locale.getDefault()).compareTo(second.name.toLowerCase(Locale.getDefault()))
```

Comment:
Replace with extension function:
```suggestion
            first.name.compareTo(second.name, ignoreCase = true)
```